src/Symfony/Component/Asset/PathPackage.php

```diff
@@ -58,7 +58,7 @@ public function getUrl($path)
 
         $versionedPath = $this->getVersionStrategy()->applyVersion($path);
 
-        if ($this->isAbsoluteUrl($versionedPath)) {
+        if ($this->isAbsoluteUrl($versionedPath) || 0 === strpos($versionedPath, '/')) {
```

Comment:
using `'/' === $versionedPath[0]` is even faster (and already used above). We just need to handle the case of the empty string.
strpos still requires looking at the whole string for the common case of needing the base path